backend/index.ts

```diff
@@ -1,12 +1,82 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+
 const app = express();
+app.use(bodyParser.json());
+
+// Connect to MongoDB
+mongoose.connect(
+  'mongodb://localhost/mydatabase',
+  { useNewUrlParser: true }
+);
+
+const itemSchema = new mongoose.Schema({
+  name: String,
+  description: String,
+});
 
-const port = process.env.PORT || 3001;
+const Item = mongoose.model('peoples', itemSchema);
 
-app.get('/', (req, res) => {
-  res.send('Dummy API');
+app.post('/add-item', async (req, res) => {
+  try {
+    const { name, description } = req.body;
+    const newItem = new Item({ name, description });
+    await newItem.save();
+    res.status(201).json({ message: 'Item added successfully', item: newItem });
+  } catch (error) {
+    res.status(500).json({ message: 'Error adding item', error: error.message });
+  }
 });
 
-app.listen(port, () => {
-  console.log(`Server is running on port ${port}`);
-});
\ No newline at end of file
+app.put('/update-item/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const { name, description } = req.body;
+
+    const updatedItem = await Item.findByIdAndUpdate(
+      id,
+      { $set: { name, description } },
+      { new: true }
+    );
+
+    if (!updatedItem) {
+      return res.status(404).json({ message: 'Item not found' });
+    }
+
+    res.json({ message: 'Item updated successfully', item: updatedItem });
+  } catch (error) {
+    res.status(500).json({ message: 'Error updating item', error: error.message });
+  }
+});
+
+app.get('/get', async (req,res)=> {
+      try{
+        const allitems = await Item.find();
+        res.json(allitems);
+      }catch(error){
+        res.status(500).json({message: 'error fetching items', error: error.message});
+      }
+})
+
+app.delete('/remove/:id', async (req,res)=> {
```

Comment:
This is nice. But ideally we donot want to delete anything from database. 
Can we add a new attribute that says isActive and put that flag to false. 

This way, if the user wants to restore his deleted blog, he can always come back. 

In all the API's we write at Amazon or old companies we don't delete anything. We just toggle some flag.